data/scm_plugins.yml

```diff
@@ -51,3 +51,12 @@
   releases_url: https://github.com/cnenning/go-artifactory-scm-plugin/releases
   github_stars: http://ghbtns.com/github-btn.html?user=cnenning&repo=go-artifactory-scm-plugin&type=watch&count=true
   first_available_date: 10/2016
+
+- name: Git Path Material
+  description: Plugin to introduce a material, that watches on a sub-directory of git repository.
+  readmore_url: https://github.com/TWChennai/gocd-git-path-material-plugin
+  author_url: https://github.com/TWChennai
+  author_name: ThoughtWorks Inc.
```

Comment:
`ThoughtWorks` or `ThoughtWorks Inc.` on this page would normally be tagged to officially maintained and supported plugins. Could you please update the author name to remove the ambiguity.